### ISSUE TITLE
[Feature] Better management of  "old" DSPCases

### ISSUE THREAD

**Issue by jocdiazmuic** [DualSPHysics/DesignSPHysics#13]
Hello,

I want to point out a situation that is currently happening among some coworkers dealing with the new version of DesignSPHysics.

**Current state**
The current behaviour of DesignsSPHysics treating "old" DSPHCases (casedata.dsphdata) is closing entirely the case  after no other option than "ok". 

**Issues**
This behaviour currently affects dozens of cases that are basically useless due to a point release update and forces the user to rollback to an outdated version.

**Proposed state**
There should be a: standalone Tool that can "port" as much as possible of information (parameters, geometry...) to the new version  OR opening the project only with the "compatible" features and allowing the user to save-as a new compatible DSPHCase with the rescued information and maybe a log of incompatible stuff for the user to track and correct.

Thank you,

**Comment by zlotny** [comment]
Hi there. This is definitely a thing we want and will do in the future. However DesignSPHysics is still under development and under heavy structural changes.

Ensuring compatibility within different versions of the tool is complicated and we're going to change all the internal data structure soon. Once this is done we'll try to look a bit more into this. However, expect this kind of inconsistency on versions 0.X until the first 1.0 version comes out.

So in summary once the new memory and saving structure is implemented this will have a bit more relevance and should work better. 

Thanks for your patience.
